packages/app/src/specs/SpecsListRowItem.vue

```diff
@@ -20,12 +20,14 @@
       >
         <div
           data-cy="specs-list-row-git-info"
+          class="group"
```

Comment:
These do not seem to do anything? `group` is used with `group-*` (like `group-hover`) which I don't see here.

Can we remove these?